docs/section19/19-22.md

```diff
@@ -0,0 +1,51 @@
+- Check if B-RMR is modeled with at least one air-side system that is of type 9 or 10, if yes then carry on, if no then NOT_APPLICABLE for the project: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_hvac_system_dict.keys() for target_system_type in target_system_types):` 
+    This segment identifies applicable zones (outcome for inapplicable zones will be NOT_APPLICABLE)
+    - For each Zone in the B_RMI: `for zone in B_RMI...Zone:` 
+        - Reset the zone_served_by_sys_9_or_10 boolean variable to FALSE: `zone_served_by_sys_9_or_10 = FALSE`  
+        - Reset zone_hvac_has_non_mech_cooling boolean variable to false: `zone_hvac_has_non_mech_cooling = False`  
+        - Get the analogous zone object in the P_RMI: `zone_p = match_data_element(P_RMR,Zone,zone.id)` 
+        - Get list of hvac systems serving this zone in the proposed design model: `hvac_sys_list_p = Get_list_hvac_systems_associated_with_zone(P_RMI, zone_p)`  
+        - Check if any of the hvac systems serving the zone have non-mechanical cooling as the cooling_system type: `for hvac_p in hvac_sys_list_P:`  
+            - Check if the system has non-mechanical cooling: `if hvac_p.cooling_system.cooling_system_options == "NON_MECHANICAL": zone_hvac_has_non_mech_cooling = True`  
+        - Check if the zone has non-mechanical cooling in the proposed design defined at the zone level or if zone_hvac_has_non_mech_cooling equals true, if false for both then the zone is NOT_APPLICABLE, otherwise carry on: `If (zone_p.non_mechanical_cooling_fan_airflow != Null AND zone_p.non_mechanical_cooling_fan_airflow != 0) or zone_hvac_has_non_mech_cooling == True:`  
+            - Get list of the HVAC systems associated with the zone in the baseline: `hvac_sys_list_b = Get_list_hvac_systems_associated_with_zone(B_RMI, zone)`  
+            - Check if any are of system 9 or 10: `for hvac in hvac_sys_list_b:`  
+                - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+                - Check if system type is of an applicable system type:  `If (baseline_system_type_compare(sys_type, HVAC_SYS.SYS_9, false) == TRUE OR baseline_system_type_compare(sys_type, HVAC_SYS.SYS_10, false) == TRUE): zone_served_by_sys_9_or_10 = TRUE`    
+                
+            - **Rule Assertion:** 
+            - Case 1: The zone has non-mechanical cooling in the proposed design and the zone is served by either system 9 or 10 in the baseline then outcome is UNDETERMINED: `if zone_served_by_sys_9_or_10 == True: outcome = "UNDETERMINED" and raise_message "<Insert zone.id> has non-mechanical cooling in the proposed design and the zone is served by either system 9 or 10 in the baseline, conduct a manual check that the baseline building design includes a separate fan to provide nonmechanical cooling, sized and controlled the same as the proposed design."`  
```

Comment:
Maybe we need a check here for if hvac_p.cooling_system.cooling_system_options == NONE or hvac_p.cooling_system.cooling_system_options == OTHER. I'm assuming the outcome would still be 'UNDETERMINED'?

Reply:
@supriyagoel I am not following, can you please explain the reason for checking this?

Reply:
@claperle i guess my comment is for the check in line 38. We check if cooling = nmc, my comment was suggesting an UNDETERMINED outcome for cooling_system_option = None or Other as well.
I guess the outcome would be undetermined irrespective of that check though, so I'll mark this as readytomerge